docs/DeveloperGuide.adoc

```diff
@@ -98,7 +100,7 @@ image::LogicClassDiagram.png[]
 *API* :
 link:{repoURL}/src/main/java/seedu/address/logic/Logic.java[`Logic.java`]
```

Comment:
What's the difference between this diagram and the diagram under the architecture part? Seems like it's repeated..